iACC/Helpers.swift

```diff
@@ -1,20 +1,59 @@
-//	
+//
 // Copyright © Essential Developer. All rights reserved.
 //
 
 import UIKit
 
 enum Formatters {
-	static var date = DateFormatter()
-	static var number = NumberFormatter()
+    static var date = DateFormatter()
+    static var number = NumberFormatter()
 }
 
 extension UIViewController {
-	var presenterVC: UIViewController {
-		parent?.presenterVC ?? parent ?? self
-	}
+    var presenterVC: UIViewController {
+        parent?.presenterVC ?? parent ?? self
+    }
+    
+    func select(friend: Friend) {
+        let vc = FriendDetailsViewController()
+        vc.friend = friend
+        show(vc, sender: self)
+    }
+    
+    func select(card: Card) {
+        let vc = CardDetailsViewController()
+        vc.card = card
+        show(vc, sender: self)    }
+    
+    func select(transfer: Transfer) {
+        let vc = TransferDetailsViewController()
+        vc.transfer = transfer
+        show(vc, sender: self)    }
```

Comment:
Fix end brackets here and on line 26. 
Also maybe move those methods to its corresponding modules.